x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_notify_when.tsx

```diff
@@ -62,22 +62,22 @@ const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<AlertNotifyWhenType>> = [
     inputDisplay: i18n.translate(
       'xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onActiveAlert.display',
       {
-        defaultMessage: 'Every time rule is active',
+        defaultMessage: 'Every time alert is active',
       }
     ),
     'data-test-subj': 'onActiveAlert',
     dropdownDisplay: (
       <Fragment>
         <strong>
           <FormattedMessage
-            defaultMessage="Every time rule is active"
+            defaultMessage="Every time alert is active"
             id="xpack.triggersActionsUI.sections.alertForm.alertNotifyWhen.onActiveAlert.label"
           />
         </strong>
         <EuiText size="s" color="subdued">
           <p>
             <FormattedMessage
-              defaultMessage="Actions run with every active rule interval."
+              defaultMessage="Actions run when the alert is active."
```

Comment:
Wondering about this one 🤔 

Should it perhaps be the following?
```suggestion
              defaultMessage="Actions run at each rule interval as long as the alert remains active."
```

Reply:
`Actions run at each rule interval that the alert is active.` ?

Reply:
Any of these?

Actions repeat at the rule interval when the alert is active
Actions repeat at regular intervals when the alert is active
When the alert is active, actions repeat based on the rule interval

Reply:
Thanks @gchaps! I think `Actions repeat at the rule interval when the alert is active` makes sense